core/src/main/kotlin/com/google/container/tools/PluginInfo.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor
+import com.intellij.ide.plugins.PluginManager
+import com.intellij.openapi.extensions.PluginId
+
+const val CONTAINER_TOOLS_PLUGIN_ID = "com.google.container.tools"
+const val UNKNOWN_VERSION = "N/A"
```

Comment:
sorry my comment wasn't very clear at all. I meant that instead of "N/A" how about making the value "UNKNOWN" or "unknown"? I was actually fine with the variable name.

Reply:
Ah I didn't read well enough. Yes, we can do that, let me change the val name back.